web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig

```diff
@@ -73,6 +73,11 @@ not node.isPublished() ? 'is-unpublished',
 view_mode ? view_mode|clean_class,
 'clearfix',
 ] %}
+{% set use_teaser = FALSE %}
+{% if node.field_teaser[0].value is not empty %}
+  {% set use_teaser = TRUE %}
+{% endif %}
```

Comment:
I'am not strong in twig land... but would this not normally be in an preprocess function or the like i D8?

Reply:
The same question in the other `spot` twig files?

Reply:
I'm not sure there d8 best practices that determine if a variable should be set in the twig layer or a preprocess function.